cumulus/polkadot-parachain/src/chain_spec/contracts.rs

```diff
@@ -14,13 +14,11 @@
 // You should have received a copy of the GNU General Public License
 // along with Cumulus.  If not, see <http://www.gnu.org/licenses/>.
 
-use crate::chain_spec::{
```

Comment:
@athei is this parachain even needed anymore? perhaps we should remove it.

Reply:
We are working on a new testnet for for pallet_revive, I think it's safe to get rid of this one now

Reply:
Okay, something to do in a follow-up then @serban300 :)

Reply:
Opened PR https://github.com/paritytech/polkadot-sdk/pull/5471 for this